hosted/json-schemas/balance.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://tbdex.dev/offering.schema.json",
+    "definitions": {
+      "Balance": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
```

Comment:
Maybe also an optional "name" or "description" field. We can also discuss further in another PR if you'd prefer to close this out ASAP.

Reply:
Do you think there's more description that would be useful beyond the currency and amount?